plugins/CorePluginsAdmin/lang/en.json

```diff
@@ -83,6 +83,7 @@
         "Version": "Version",
         "ViewAllMarketplacePlugins": "View all Marketplace plugins",
         "WeCouldNotLoadThePluginAsItHasMissingDependencies": "The plugin %1$s could not be loaded as it has missing dependencies: %2$s",
-        "Websites": "Websites"
+        "Websites": "Websites",
+        "UpdateSelected": "Updateg Selected"
```

Comment:
Now it's another typo 🙈

Reply:
argh, i think that was part of the git command to commit :)